### ISSUE TITLE
All integration tests are failing

### ISSUE THREAD

**Issue by gtristan** [apache/buildstream#1549]
It seems that lately, *all* integration tests (where we actually run commands in the sandbox) are failing.

From an [example CI workflow](https://github.com/apache/buildstream/runs/4648483172?check_suite_focus=true):

```
Failure Summary
    autotools/amhello.bst:
    [00:00:06][e522785a][   build:autotools/amhello.bst         ] FAILURE Running commands
    [00:00:06][e522785a][   build:autotools/amhello.bst         ] FAILURE Command failed

        Printing the last 20 lines from log file:
        /home/testuser/buildstream/.tox/py310-nocover/tmp/test_pullbuildtrees0/cache/logs/test/autotools-amhello/e522785a-build.3478.log
        ======================================================================
        sh: write error: I/O error
        cat: write error: I/O error
        sh: write error: I/O error
        Command './configure --prefix=/usr \
        --exec-prefix=/usr \
        --bindir=/usr/bin \
        --sbindir=/usr/sbin \
        --sysconfdir=/etc \
        --datadir=/usr/share \
        --includedir=/usr/include \
        --libdir=/usr/lib \
        --libexecdir=/usr/libexec \
        --localstatedir=/var \
        --sharedstatedir=/usr/com \
        --mandir=/usr/share/man \
        --infodir=/usr/share/info   ' failed with exitcode 2
        [00:00:06] FAILURE autotools/amhello.bst: Running commands
        [--:--:--] START   [e522785a] autotools/amhello.bst: Caching artifact
        [00:00:00] SUCCESS [e522785a] autotools/amhello.bst: Caching artifact
        [00:00:06] FAILURE [e522785a] autotools/amhello.bst: Command failed
        ======================================================================
```

Seems we are getting these weird I/O errors.

We should:
* Find the root cause of this
* Have BuildStream make an early exit when it finds it is running on a host that is incapable of running commands properly
* Fix the infra to properly support builds

**Comment by juergbi** [comment]
This may be fixed by updating buildbox. We should merge https://github.com/apache/buildstream/pull/1544. buildbox-fuse had a bug that only causes issues on Linux 5.13+ (or possibly on older versions with backports). I.e. this issue may have been triggered by a kernel upgrade in the CI infrastructure.

**Comment by abderrahim** [comment]
Yes, this is the reason I opened #1544.

**Comment by gtristan** [comment]
Nice thanks ! This is indeed solved by #1544, closing.
